x-pack/legacy/plugins/infra/public/containers/logs/log_analysis/api/ml_setup_module_api.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as rt from 'io-ts';
+import { kfetch } from 'ui/kfetch';
+
+import { getJobIdPrefix } from '../../../../../common/log_analysis';
+import { throwErrors, createPlainError } from '../../../../../common/runtime_types';
+
+export const callSetupMlModuleAPI = async (
+  spaceId: string,
+  sourceId: string,
+  indexPattern: string,
+  timeField: string,
+  bucketSpan: number
+) => {
+  const response = await kfetch({
+    method: 'POST',
+    pathname: '/api/ml/modules/setup',
```

Comment:
This doesn't need the `/api/ml/modules/setup/{id}` at the end? e.g. I expected this URL to be `/api/ml/modules/setup/logs_ui_analysis`, similar to how APM calls this module API: 

https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/apm/public/services/rest/ml.ts#L55

Reply:
Looks like a mistake, `/api/ml/modules/setup/{id}` would be correct 👍